log.go

```diff
@@ -155,6 +173,10 @@ func (l Logger) Output(w io.Writer) Logger {
 	l2 := New(w)
 	l2.level = l.level
 	l2.sampler = l.sampler
+	if len(l.hooks) > 0 {
+		l2.hooks = make([]Hook, len(l.hooks), cap(l.hooks))
+		copy(l2.hooks, l.hooks)
```

Comment:
`append` should be slightly faster in this context (with len=0 on prev line indeed).

Reply:
Yeah, using append now, and without using make first.